src/scarlet/license_finder/tool.clj

```diff
@@ -0,0 +1,16 @@
+(ns scarlet.license-finder.tool
+  (:require
+   [scarlet.license-finder.core :as core]
+   [scarlet.license-finder.report :as report]))
+
+(defn report
+  [{:keys [project format path]
+    :or   {project "./deps.edn"
+           format :csv
+           path "./target/licenses/deps.csv"}
```

Comment:
In README there's an example showing `:path :stdout` kv-pair to generate a report printed to standard output.
I thinks it's ok for now. Later, if options get more complex, a proper documentation for all the available options may be needed.